frontend/src/metabase/public/components/EmbedFrame.jsx

```diff
@@ -43,12 +43,13 @@ class EmbedFrame extends Component {
     } = this.props;
     const { innerScroll } = this.state;
 
-    const showFooter = !MetabaseSettings.hideEmbedBranding() || actionButtons;
-
-    const { bordered, titled, theme, hide_parameters } = {
+    const { bordered, titled, theme, hide_parameters, show_download_button } = {
```

Comment:
Maybe `hide_download_button` to match `hide_parameters`? Or add `show_download_button: true` to `DEFAULT_OPTIONS` to not break existing embeds. It would rename the param fwiw.